docs/src/content/docs/guides/recurring-subscription-incoming-amount.mdx

```diff
@@ -0,0 +1,548 @@
+      },
+      {
+        interact_ref: interactRef
+      }
+    )
+    ```
+  </TabItem>
+</Tabs>
+
+<details>
+  <summary>Example response</summary>
+The following shows an example response from the customer's wallet provider.
+
+```json wrap
+{
+  "access_token": {
+    "value": "...", // final access token required before creating outgoing payments
+    "manage": "https://auth.cloudninebank.example.com/token/{...}", // management uri for access token
+    "access": [
+      {
+        "type": "outgoing-payment",
+        "actions": ["create", "read"],
+        "identifier": "https://cloudninebank.example.com/customer",
+        "limits": {
+          "debitAmount": {
+            "assetCode": "USD",
+            "assetScale": 2,
+            "value": "1500"
+          },
+          "interval": "R12/2025-10-14T00:00:00Z/P1M"
+        }
+      }
+    ]
+  },
+  "continue": {
+    "access_token": {
+      "value": "..." // access token for continuing the request
+    },
+    "uri": "https://auth.cloudninebank.example.com/continue/{...}" // continuation request uri
+  }
+}
+```
+
+</details>
+
+### 10. Request the creation of an outgoing payment resource
+
+Use the access token returned in Step 9 to call the <Badge text="POST" variant="success" /> [Create Outgoing Payment API](/apis/resource-server/operations/create-outgoing-payment).
+
+Include the `quoteId` from the quote created in Step 5.
+
+<Tabs syncKey="lang">
+<TabItem label='TypeScript/JavaScript' icon='seti:javascript'>
+  ```ts wrap
+  const customerOutgoingPayment = await client.outgoingPayment.create(
+    {
+      url: customerWalletAddress.resourceServer,
+      accessToken: customerOutgoingPaymentGrant.access_token.value
+    },
+    {
+      walletAddress: customerWalletAddress.id,
+      quoteId: customerQuote.id
+    }
+  )
+```
+</TabItem>
+</Tabs>
+
+<details>
+<summary>Example response</summary>
+The following shows an example response from the customer's wallet provider.
+
+```json wrap
+{
+  "id": "https://cloudninebank.example.com/outgoing-payments/{...}", // url identifying the outgoing payment
+  "walletAddress": "https://cloudninebank.example.com/customer",
+  "receiver": "https://happylifebank.example.com/incoming-payments/{...}", // url of the incoming payment being paid
+  "debitAmount": {
+    "value": "1500",
+    "assetCode": "USD",
+    "assetScale": 2
+  },
+  "receiveAmount": {
+    "value": "1500",
+    "assetCode": "USD",
+    "assetScale": 2
+  },
+  "sentAmount": {
+    "value": "0",
+    "assetCode": "USD",
+    "assetScale": 2
+  },
+  "createdAt": "2025-10-14T00:00:54.52Z"
+}
+```
+
+</details>
+
+The first of the twelve recurring subscription payments is now set up. At the next interval (one month from now), repeat Steps 3, 5, and 10. You don't need to request new outgoing or incoming payment grants because the original grants are still valid for the remaining billing periods.
```

Comment:
Do you need/not need a new quote grant? If yes, add that step to the list. If no, then maybe update the paragraph to "You don't need to request new grants because...".